lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -14,21 +14,20 @@ local CENTER_UPDATE_INTERVAL = customSpawnConfigForMap.centerUpdateInterval or C
 local IGNORE_BUILDERS = CFCRandomSpawn.Config.IGNORE_BUILDERS
 local CENTER_UPDATE_ON_RESPAWN = CENTER_UPDATE_INTERVAL <= 0
 
-local function calculatePvpCenters()
-    if not pvpCenters or not pvpCenters[1] then
-        local avgSum = Vector()
-        local count = #customSpawnsForMap
-
-        for _, spawn in pairs( customSpawnsForMap ) do
-            avgSum = avgSum + spawn.spawnPos
-        end
-
-        pvpCenters = pvpCenters or {}
-        pvpCenters[1] = {
-            centerPos = avgSum / count
-        }
+local DYNAMIC_CENTER_MINSIZE = 1750 -- getDynamicPvpCenter starts at this radius
+local DYNAMIC_CENTER_MAXSIZE = 3750 -- no bigger than this radius
+local DYNAMIC_CENTER_MINSPAWNS = 10 -- getDynamicPvpCenter needs at least this many spawns inside the radius
+local DYNAMIC_CENTER_MAXSPAWNS = 30 -- max possible spawns
+local DYNAMIC_CENTER_SPAWNCOUNTMATCHPVPERS = true -- pvp center gets bigger when more people are pvping
+local DYNAMIC_CENTER_IMPERFECT = true -- throw a bit of randomness in, makes pvp less stiff.
+
+local function defaultPvpCenter()
+    if pvpCenters[1] then return pvpCenters[1] end
```

Comment:
```suggestion
local function defaultPvpCenter()
    return pvpCenters[1]
```
could simplify to this

Reply:
👍